src/api/public/apidocs-new/paths/my_notifications_id.yaml

```diff
@@ -0,0 +1,31 @@
+put:
+  summary: Toggle the read/unread state of a notification.
+  description: |
+    Toggle the boolean value of the `delivered` attribute of a notification, which
```

Comment:
This is referring to the internal representation of notifications, but it's not useful to users. I would suggest something like this summary and description:

Summary: Mark a notification as read/unread
Description: If a notification is unread, it will be marked as read. If a notification is read, it will be marked as unread.

Reply:
@dmarcoux Agree, applied